marketplace/netlify-build/src/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { init } from 'contentful-ui-extensions-sdk';
+import { createPubSub } from './pubnub-client';
+import { Select, Option } from '@contentful/forma-36-react-components';
+import NeflifySideBarBuildButton from './NetlifySideBarBuildButton';
+
+function createSiteStructure(parameters = {}) {
+  const buildHooks = (parameters.buildHookUrls || '').split(',');
```

Comment:
I would do something like this:

```js
const readCsvParam = csv => (csv || '')
  // split by comma
  .split(',')
  // remove whitespace around the value
  .map(val => val.trim())
  // filter out empty values in case of double comma `,,`
  .filter(val => val.length > 0)

const buildHooks = readCsvParam(parameters.buildHookUrls)
// ...
```

Reply:
Good call, will change.